src/main/java/com/devteam/sandboxbasedapp/persistence/entity/ParkingSpaceEntity.java

```diff
@@ -0,0 +1,26 @@
+package com.devteam.sandboxbasedapp.persistence.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "parking_spaces", schema = "public")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class ParkingSpaceEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "status")
+    private String status;
```

Comment:
Let's create a java enum for the statuses

Reply:
Done.